packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -46,3 +48,41 @@ where
             .change_context(QueryError)
     }
 }
+
+#[async_trait]
+impl<C: AsClient, T> crud::Read<'_, (), T> for PostgresStore<C>
+where
+    for<'de> T: OntologyDatabaseType + Deserialize<'de> + Send,
+{
+    type Output = Vec<T>;
+
+    async fn get(&self, _: ()) -> Result<Self::Output, QueryError> {
+        // SAFETY: We insert a table name here, but `T::table()` is only accessible from within this
+        //   module.
+        let row_stream = self
+            .as_client()
+            .query_raw(
+                &format!(
+                    r#"
+                    SELECT schema
+                    FROM {};
```

Comment:
Is it intentional to retrieve all schemas, including all versions here? Or did we mean to only get the latest version of every schema?

Reply:
Resolving as it's addressed in 96936ce3f1675753a4f883220ddfb51784ca1f4f :)